pkg/forklift-api/services/tls-certificate.go

```diff
@@ -21,6 +22,7 @@ func serveTlsCertificate(resp http.ResponseWriter, req *http.Request, client cli
 				Type:  "CERTIFICATE",
 				Bytes: cacert.Raw,
 			})
+			encoded = bluemonday.UGCPolicy().SanitizeBytes(encoded)
```

Comment:
can't we sanitize the certificate using x509.ParseCertificate instead?

Reply:
but using `x509.ParseCertificate` will take e.g the `encoded` and generate a new cert from it, where i need to re-encode?

Reply:
right, I didn't mean to parse the encoded block but cacert.Raw - IIUC the risk is in getting malicious code there, no?